support-frontend/assets/pages/paper-subscription-checkout/helpers/__tests__/voucherDeliveryDays.test.ts

```diff
@@ -0,0 +1,69 @@
+// ----- Imports ----- //
```

Comment:
Think `getVoucherDays` is only used in the old `paperCheckoutForm.ts` which we have not deprecated yet so needs to stay along with this test